doc/releases/changelog-0.23.0.md

```diff
@@ -583,18 +594,22 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
 * Optimizers only consider a variable trainable if they have `requires_grad = True`.
   [(#2381)](https://github.com/PennyLaneAI/pennylane/pull/2381)
 
-* Fixes a bug with `qml.expval`, `qml.var`, `qml.state` and
+* Fixed a bug with `qml.expval`, `qml.var`, `qml.state` and
   `qml.probs` (when `qml.probs` is the only measurement) where the `dtype`
   specified on the device did not match the `dtype` of the QNode output.
   [(#2367)](https://github.com/PennyLaneAI/pennylane/pull/2367)
 
-* Fixes cases with `qml.measure` where unexpected operations were added to the
-  circuit.
-  [(#2328)](https://github.com/PennyLaneAI/pennylane/pull/2328)
+* Fixed a bug where the output shapes from batch transforms are inconsistent
+  with the QNode output shape.
+  [(#2215)](https://github.com/PennyLaneAI/pennylane/pull/2215)
+
+* Fixed a bug caused by the squeezing in `qml.gradients.param_shift_hessian`.
+  [(#2215)](https://github.com/PennyLaneAI/pennylane/pull/2215)
 
-* Fixes a bug in which the `expval`/`var` of a `Tensor(Observable)` would depend on the order
+* Fixed a bug in which the `expval`/`var` of a `Tensor(Observable)` would depend on the order
   in which the observable is defined:
-  ```python
+  [(#2276)](https://github.com/PennyLaneAI/pennylane/pull/2276)
+  ```pycon
   @qml.qnode(dev)
```

Comment:
@antalszava you'll need to write the entire block in pycon syntax:

````markdown
```pycon
>>> @qml.qnode(dev)
... def circ(op):
...   qml.RX(0.12, wires=0)
...   qml.RX(1.34, wires=1)
...   qml.RX(3.67, wires=2)
...   return qml.expval(op)
>>> op1 = qml.Identity(wires=0) @ qml.Identity(wires=1) @ qml.PauliZ(wires=2)
>>> op2 = qml.PauliZ(wires=2) @ qml.Identity(wires=0) @ qml.Identity(wires=1)
>>> print(circ(op1), circ(op2))
-0.8636111153905662 -0.8636111153905662
```
````